server/models/Message.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const messageSchema = new mongoose.Schema({
+    conversationId: {
```

Comment:
I don't think we need conversation reference here

Reply:
Removed conversation ref from Model and conversation Id from controller